test/server/admin/config_dump_handler_test.cc

```diff
@@ -16,13 +16,25 @@ INSTANTIATE_TEST_SUITE_P(IpVersions, AdminInstanceTest,
 
 // helper method for adding host's info
 void addHostInfo(NiceMock<Upstream::MockHost>& host, const std::string& hostname,
-                 const std::string& address_url, envoy::config::core::v3::Locality& locality,
+                 const std::vector<std::string> addresses_url_list,
```

Comment:
nit: I think it may be better to separate the `address_url` from the list of addresses (i.e., add an additional `additional_addresses_url`), as the expected output is different - the `address_url` will be in its own field, and the list will be in their own field. (keeping the same separation)

Reply:
Sure, I will update it :+1:

Reply:
Done, changed back to having two separate fields: address_url and additional_addresses_url.